README.md

```diff
@@ -1,2 +1,77 @@
 # catalog-of-my-things-ruby-capstone
-Catalog-of-my-things In this project, you will create a console app that will help you to keep a record of different types of things you won: books, music albums, movies, and games. Everything will be based on the UML class diagram presented below. The data will be stored in JSON files but you will also prepare a database with tables structure analogical to your program's class structure
+
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+Catalog-of-my-things In this project, we created a console app that will help you to keep a record of different types of things you won: books, music albums, movies, and games. Everything will be based on the UML class diagram presented below. The data will be stored in JSON files and will also prepare a database with tables structure analogical to your program's class structure
+## Built With
+- Ruby, PostgreSQL
+- Rspec
+## Getting Started
+To get a local copy up and running, follow the steps in Setup below.
+
+### Prerequisites
+Basic knowledge of Ruby and SQL is required
+
+### Setup
+If you installed git you can clone the code to your machine, or download a ZIP of all the files directly.
+
+[Download the ZIP from this location](https://github.com/topeogunleye/catalog-of-my-things-ruby-capstone/archive/refs/heads/main.zip), or run the following [git](https://git-scm.com/downloads) command to clone the files to your machine:
+
+```bash
+git clone https://github.com/topeogunleye/catalog-of-my-things-ruby-capstone.git
+```
+
+- Once the files are on your machine, open the _catalog-of-my-things-ruby-capstone_ folder in [Visual Studio Code](https://code.visualstudio.com/), and follow the steps in Installation & Set Up
+
+## 🛠 Installation & Set Up
+
+1. Install Ruby
+
+   ```sh
+   sudo apt update && sudo apt install -y ruby-full && ruby --version
+   ```
+
+2. Run files locally
+
+   ```sh
+   ruby *.rb
+   ```
```

Comment:
- [ ] Let's make this section more descriptive guys. Indeed I did have to find how to run your app and figured it out by myself... Your readme should provide all essential information about the project. In this case, you may split this section in two-part: 

1. Run App
   ```rb
   ruby main.rb
   ```
2. Run tests
   ```rb
   rspec spec --format documentation
   ```